### PR TITLE
Adding Danger xcode-sumary

### PR DESCRIPTION
**Pull request by LucianoPAlmeida**
## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [ ] New extensions are written in Swift 4.
- [ ] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+.
- [ ] I have added tests for new extensions, and they passed.
- [ ] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [ ] All extensions are declared as **public**.
- [ ] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

File: Dangerfile [hunk 1]
```diff
@@ -10,15 +10,20 @@ no_changelog_entry = !git.modified_files.include?("CHANGELOG.md")
...
+    warn("Consider adding tests for new extensions or updating existing tests for a modified SwifterSwift extension")
 end
 
 if source_changes_exist && no_changelog_entry
-    warn(“The source files have been modified. Please consider adding a CHANGELOG entry if necessary.“)
+    warn("The source files have been modified. Please consider adding a CHANGELOG entry if necessary.")
 end
 
+#Xcode summary
+xcode_summary.report 'xcodebuild-ios.json'
```
Comment (SD10) [inline Dangerfile]: We should also put these at the very end of the `Dangerfile`

Comment (LucianoPAlmeida) [reply Dangerfile]: Sure :))

**Comment by SD10** [comment]
@LucianoPAlmeida Awesome work here, I restarted the builds for you. I'm not sure if they are failing due to the warnings from Xcode summary or something else

**Comment by LucianoPAlmeida** [comment]
@SD10 I think is the pod lib lint that doesn't allow warnings, so the build is failing on that
There was also a test breaking before on the push build, but not in the pr one ¯\_(ツ)_/¯ I have to check that as soon as possible.

**Comment by LucianoPAlmeida** [comment]
But I think the xcode_sumary is working nicely now, just need to do a few adjustments \o/

File: Sources/Extensions/SwiftStdlib/ArrayExtensions.swift [hunk 2]
```diff
@@ -104,6 +104,9 @@ public extension Array {
 	/// - Returns: The new rotated array
 	public func rotated(by places: Int) -> [Element] {
 		//Inspired by: https://ruby-doc.org/core-2.2.0/Array.html#method-i-rotate
+        
+        var random: Int = 0
```
[the remark this answers is not in the document]
Comment (LucianoPAlmeida) [reply Sources/Extensions/SwiftStdlib/ArrayExtensions.swift]: Yeap, I just put it there to see the warning of xcode_summary as a test 🤣

**Comment by SD10** [comment]
@LucianoPAlmeida I just did it via GitHub for you

**Comment by LucianoPAlmeida** [comment]
@SD10 If the build passes I think it's good to go 👍

**Comment by LucianoPAlmeida** [comment]
That's nice 👍  
Thank's man :))

**Pull request merged**